src/main/java/by/yahor_kulesh/model/users/Client.java

```diff
@@ -0,0 +1,21 @@
+package by.yahor_kulesh.model.users;
+
+import by.yahor_kulesh.model.tickets.Ticket;
+
+import java.util.HashSet;
+import java.util.Set;
+
+public class Client extends User {
+    private final Set<Ticket> tickets = new HashSet<>();
+
+    public void getTicket(Ticket ticket) {
+        System.out.println("Client got ticket:" + ticket.getId() + "\n");
+        tickets.add(ticket);
+        this.print();
```

Comment:
do not mix logic. single method single responsability. if you need to print smth - call it as next command in service/app